src/backend/distributed/commands/function.c

```diff
@@ -192,6 +202,26 @@ create_distributed_function(PG_FUNCTION_ARGS)
 
 	ObjectAddressSet(functionAddress, ProcedureRelationId, funcOid);
 
+	DistObjectCacheEntry *cacheEntry =
+		LookupDistObjectCacheEntry(ProcedureRelationId, funcOid, InvalidOid);
+
+	if (cacheEntry != NULL && IsObjectDistributed(&functionAddress) &&
+		IsCreateDistributedFunctionCallIdempotent(cacheEntry,
+												  distributionArgumentName,
+												  colocateWithTableNameDefault,
+												  forceDelegationAddress))
+	{
+		char *schemaName = get_namespace_name(get_func_namespace(funcOid));
+		char *functionName = get_func_name(funcOid);
+		char *qualifiedName = quote_qualified_identifier(schemaName, functionName);
+		ereport(NOTICE, (errmsg("function %s is already distributed", qualifiedName),
+						 errdetail("Citus distributes functions with CREATE "
+								   "(OR REPLACE) FUNCTION command"),
+						 errhint("To delegate the function, please provide parameters "
```

Comment:
I think we can write the name of the parameters here, as table name is also a parameter it can be confusing.